Mage.Common/src/main/java/mage/remote/messages/requests/GetTournamentChatIdRequest.java

```diff
@@ -0,0 +1,25 @@
+package mage.remote.messages.requests;
```

Comment:
This and the other 100 files that got created in the callback, requests, and reponses folder. Where did the code for them come from?

Reply:
It's a combination of the code from Network Upgrade and my own work.